### PR TITLE
Improve the ui on tablets/small screen devices

### PR DESCRIPTION
**Pull request by jor3l**
fixes CM-789

testplan: 
- using commons, preview a document on a tablet, notice the preview uses less space (reduced paddings/margins) and the prev/next buttons work. Compare with prod if possible.

### CODE REVIEW

**Comment by ext4se** [comment]
QR+1 and CR+1. Please resolve unit test fails in travis.

**Comment by ext4se** [comment]
We need to fix transitions between viewport sizes and the small view template at a later point.

File: src/CommonCartridge.js [hunk 1]
```diff
@@ -449,382 +469,398 @@ export default class CommonCartridge extends Component {
...
+                .join(" ")
+                .trim()}
+            >
+              {this.state.showcaseSingleResource === null && (
+                <nav>
+                  <input type="checkbox" id="NavTrigger" name="NavTrigger" />
+                  <Button
+                    as="label"
+                    variant="ghost"
+                    buttonRef={ref =>
```
Comment (solkimicreb) [inline src/CommonCartridge.js]: This seems like it does not belong here. Why don't you add a className='open' and for='NavTrigger' props to the Button?

File: src/CommonCartridge.js [hunk 2]
```diff
@@ -449,382 +469,398 @@ export default class CommonCartridge extends Component {
...
+                        }}
+                      >
+                        <Trans>Files ({this.state.fileResources.length})</Trans>
+                      </NavLink>
+                    )}
+                  </div>
+                  <Button
+                    as="label"
+                    variant="ghost"
+                    buttonRef={ref =>
```
Comment (solkimicreb) [inline src/CommonCartridge.js]: Same as above comment. Maybe I am missing something. I don't exactly know what buttonRef does but I suspect it's just a simple ref forwarding prop of the Button component. In that case I think this logic does not belong here.

File: src/Resource.js [hunk 3]
```diff
@@ -91,10 +91,11 @@ export default class Resource extends Component {
       ? `/module-items/${module.identifierref || module.identifier}`
       : `/resources/${module.identifierref || module.identifier}`;
 
-  renderPreviousButton = previousItem => {
+  renderPreviousButton = (previousItem, withTooltip) => {
+    let Tip = withTooltip ? Tooltip : React.Fragment;
```
Comment (solkimicreb) [inline src/Resource.js]: Cool trick, I will steal this one 😄

File: src/CommonCartridge.js (hunk 1 above)
Comment (jor3l) [reply src/CommonCartridge.js]: Is just that className does not work with instui so the workaround is to use the ref instead

File: src/CommonCartridge.js (hunk 2 above)
Comment (jor3l) [reply src/CommonCartridge.js]: Same here, in order to have attributes and classes the ref is used (its a hack)

File: src/Resource.js (hunk 3 above)
Comment (jor3l) [reply src/Resource.js]: lol 💯

**Pull request merged**